### ISSUE TITLE
TT1 blocks: use container blocks with padding controls instead of spacers

### ISSUE THREAD

**Issue by vindl** [WordPress/theme-experiments#223]
Currently we are using Spacer blocks to simulate padding within Header and Footer areas in this theme.

<img width="1680" alt="Screenshot 2021-02-23 at 14 46 26" src="https://user-images.githubusercontent.com/1182160/108852495-f7c86c80-75e5-11eb-914a-40ac4df89f42.png">

Would it make sense to replace these with Group block (or similar) that would allow us to control the padding via its block settings:

<img width="273" alt="Screenshot 2021-02-23 at 14 47 48" src="https://user-images.githubusercontent.com/1182160/108852613-1f1f3980-75e6-11eb-9706-350ad01c636c.png">

cc @kjellr

**Comment by carolinan** [comment]
I think the question is, is there any open issue about or progress on adding the padding to the template parts?

Like you said the group block is the closest block with padding that we could use, and that means adding two unnecessary wrappers inside the template part . -I much prefer the template parts because they reduce the div soup 🥣

**Comment by vindl** [comment]
> I think the question is, is there any open issue about or progress on adding the padding to the template parts?

Good point, now that you mention it I think there was an issue for that, but I'm having trouble finding it now.

**Comment by pagelab** [comment]
> Good point, now that you mention it I think there was an issue for that, but I'm having trouble finding it now.

@vindl Is this one: WordPress/Gutenberg#28845

**Comment by kjellr** [comment]
I took another look at this today, but I'm not totally sure about adding this to the template part. If we do that, we would have to add the spacing individually in each template that uses this template part. If a user were to edit the spacing in one of those templates, their change won't sync across to every other template. 

With the current spacers approach, we're setting the spacing _within_ the template part itself, which means it's defined in one place instead of many. If a user changes it, the spacing will update on all instances of that template part. 

We could of course add another wrapping element in the header/footer, and add padding to that. But that results in additional nested items, and I'm not sure that's preferable.

**Comment by vindl** [comment]
Closing this since it's no longer relevant.
